app/components/shared/events.tsx

```diff
@@ -12,13 +12,16 @@ export function EventsList({ events }: { events: Event[] }) {
 		<ul className="grid gap-6 md:grid-cols-2 lg:grid-cols-3">
 			{events.map(event => (
 				<li key={event.id}>
-					<Card className="h-full rounded-xl p-6">
-						<div className="w-full overflow-hidden rounded-lg">
+					<Card className="group h-full rounded-xl p-6">
```

Comment:
i think it will be great when we hover the card, there is a shadow around the card

Reply:
yeah, I've add the shadow for the card